src/d3a/d3a_core/export.py

```diff
@@ -16,33 +16,35 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 import csv
+import json
 import logging
-import pathlib
+import operator
 import os
+import pathlib
 import shutil
-import json
-import operator
-from typing import Dict
 from collections import namedtuple
 from copy import deepcopy
 from functools import reduce  # forward compatibility for Python 3
+from typing import Dict, Tuple, List, Mapping
 
+import d3a.constants
```

Comment:
Nit: something seems off with the order of these imports. Could you please try to optimize them again?

Reply:
https://user-images.githubusercontent.com/33482427/132870684-113678a9-7629-4450-a3f4-3e5ebbbbcec3.mov

Tried, the ordered stayed as it is now

Reply:
Could you please check your settings? I'm not sure what the issue might be, but the order you have is not correct: imports from `d3a_interface` (which is an external library) should be above the ones from `d3a`.

![image](https://user-images.githubusercontent.com/2814672/132875650-c63c2573-a71c-48ad-ab15-3c366a7f8261.png)

Reply:
![Screenshot 2021-09-10 at 18 47 54](https://user-images.githubusercontent.com/33482427/132888930-5dcc16dd-da7a-4a79-8821-913ed0e7df75.png)


I think I have the same settings as you